packages/editor/src/components/post-trash/check.js

```diff
@@ -13,10 +13,10 @@ import { GLOBAL_POST_TYPES } from '../../store/constants';
 /**
  * Wrapper component that renders its children only if the post can trashed.
  *
- * @param {Object}        props          - The component props.
- * @param {React.ReactEl} props.children - The child components to render.
+ * @param {Object}          props          - The component props.
+ * @param {React.ReactNode} props.children - The child components to render.
```

Comment:
I don't have the whole context around these changes, but do we actually need `ReactNode` instead of `ReactElement` ? In other words, do we ever need to render `children` that are  `null`, `undefined`, `boolean`, `number`, `string` ?

Just asking in case the right choice here was `ReactElement`.

If we go for `ReactElement`, we could also consider updating the return type to `ReactElement | null`

Reply:
I'm not sure, but I'm sure ReactElement is a good substitute here 👍🏻 

I'll do a follow up.